benchmark/buffers/buffer-indexof-number.js

```diff
@@ -0,0 +1,23 @@
+'use strict';
+var common = require('../common.js');
+var fs = require('fs');
+const path = require('path');
```

Comment:
Nit: inconsistent `var` and `const` usage. I suppose it was copied from `benchmark/buffers/buffer-indexof.js`.